transmart-core-db/grails-app/services/org/transmartproject/db/clinical/MultidimensionalDataResourceService.groovy

```diff
@@ -831,13 +822,46 @@ class MultidimensionalDataResourceService implements MultiDimensionalDataResourc
         }
     }
 
+    @Override List retriveHighDimDataTypes(MultiDimConstraint assayConstraint_, User user){
+
+        Constraint assayConstraint = (Constraint) assayConstraint_
```

Comment:
I followed the convention which was used by Jan in other methods in this class. Should I refactor them as well?